css/style.css

```diff
@@ -45,7 +45,7 @@ a {
   background: transparent;
   cursor: pointer;
 }
-
+/*comment her*/
 .grid {
   display: flex;
```

Comment:
I think it will make more sense if you made it grid here as you choose grid as an id label